Exam/CT/ds/src/exercice1/transform.java

```diff
@@ -0,0 +1,46 @@
+package exercice1;
+
+public class transform {
+
+	public static void main(String[] args) {
```

Comment:
Please read the instructions carefully...
No `main`! Create a Junit test instead!